app/src/main/res/values/strings.xml

```diff
@@ -252,6 +252,8 @@
 
     <!-- Devices String-->
     <string name="connect_success">Device Connected Successfully !</string>
+    <string name="auth_success">Auth credentials sent Successfully !</string>
+    <string name="wifi_success">Wifi credentials sent Successfully !</string>
```

Comment:
Remove space before '!'

Reply:
'successfully' (small s)

Reply:
not important, to be deleted anyways put only for testing purpose